packages/docs/pages/usage/create.mdx

```diff
@@ -22,6 +22,7 @@ Orama supports the following types:
 | `string[]`       | An array of strings.                                                        | `['red', 'green', 'blue']`                                                  |
 | `number[]`       | An array of numbers.                                                        | `[42, 91, 28.5]`                                                            |
 | `boolean[]`      | An array of booleans.                                                       | `[true, false, false]`                                                      |
+| `enum`           | An enum value.                                                              | `['drama', 'crime', 'war']`                                                 |
```

Comment:
@allevo having a bad day? 😂 I would use a bit more positive examples in the docs (like `['comedy', 'action', 'romance']`)